backend/src/routes/api/notebook-images/notebooksImageStreamUtils.ts

```diff
@@ -39,20 +39,20 @@ const mapImageStreamToNotebook = (is: ImageStreamKind): Notebook => ({
     is.spec.tags &&
     (JSON.parse(
       is.spec.tags[0].annotations['opendatahub.io/notebook-python-dependencies'],
-    ) as NotebookPackage[]),
+    ) as NotebookImagePackage[]),
   software:
     is.spec.tags &&
     (JSON.parse(
       is.spec.tags[0].annotations['opendatahub.io/notebook-software'],
-    ) as NotebookPackage[]),
+    ) as NotebookImagePackage[]),
   uploaded: is.metadata.creationTimestamp,
   url: is.metadata.annotations['opendatahub.io/notebook-image-url'],
   user: is.metadata.annotations['opendatahub.io/notebook-image-creator'],
 });
 
 export const getNotebooks = async (
```

Comment:
In this function, you are only getting image streams with `app.kubernetes.io/created-by=byon` label. Should we get all the image streams for KFNBC? Or we can set a query param to differentiate that?

Reply:
Right, we should definitely differentiate that, I will try to make some logic around it.

Reply:
@maroroman BTW, could you please also add the `tags` to the return object? For BYON, it may only use packages and software so it's OK to only return `is.spec.tags[0]`, but for KFNBC it should return all the tags, and users could choose different tags of the same image stream.

Reply:
After talking to @DaoDaoNoCode it looks like this api call is customized heavily for byon.  I suggested he create his own endpoint and work with @maroroman to review and modify as needed.